Sources/Rendering/OpenGL/CustomPostProcessingPass/index.js

```diff
@@ -0,0 +1,317 @@
+import macro from 'vtk.js/Sources/macro';
+import vtkOpenGLFramebuffer from 'vtk.js/Sources/Rendering/OpenGL/Framebuffer';
+import vtkRenderPass from 'vtk.js/Sources/Rendering/SceneGraph/RenderPass';
+import vtkDataArray from 'vtk.js/Sources/Common/Core/DataArray';
+import vtkHelper from 'vtk.js/Sources/Rendering/OpenGL/Helper';
+import vtkVertexArrayObject from 'vtk.js/Sources/Rendering/OpenGL/VertexArrayObject';
+
+import { Representation } from 'vtk.js/Sources/Rendering/Core/Property/Constants';
+
+const { vtkErrorMacro } = macro;
+// ----------------------------------------------------------------------------
+
+function vtkCustomPostProcessingPass(publicAPI, model) {
```

Comment:
`vtkCustomKernelPass` or `vtkConvolutionPass` would be more indicative names for what the class does.

Reply:
We agree that the chosen name is not very descriptive. What about vtk2dConvolutionPass to make it clear that it uses a 2D kernel on a 2D image?

Reply:
I agree, `vtk2DConvolutionPass` sounds good.